cmd/argo/commands/template/lint.go

```diff
@@ -77,7 +77,11 @@ func ServerSideLint(args []string, strict bool) error {
 			}
 			return nil
 		}
-		return filepath.Walk(args[0], walkFunc)
+		err := filepath.Walk(args[0], walkFunc)
+		if err != nil {
```

Comment:
test?

Reply:
Currently, without a mock APIServer we can't write the test. The test will be added in CLI test framework PR

Reply:
e2e test?

Reply:
added e2e test